setup.py

```diff
@@ -28,7 +28,7 @@
         "tqdm",
     ],
     extras_require={
-        "mujoco": ["mujoco>=2.3.0,<3.0.0", "dm_control", "numba", "opencv-python"],
+        "mujoco": ["mujoco>=3.0.0", "dm_control", "numba", "opencv-python"],
```

Comment:
Since all geom elements have contype/conaffinity set explicitly, the deprecated "collision" option has no any effect, so the updated XML file should work in the same way with both old and new mujoco versions. Probably it's not really necessary to drop mujoco 2.3.x support (at least at this point)

Reply:
Thanks for pointing this out! I will change it.